docs/reference-guides/providers/certified.md

```diff
@@ -21,6 +21,8 @@ This is a list of the officially certified CAPI Providers by Turtles. These prov
 | **AWS**         | CAPA                           | Infrastructure           | https://cluster-api-aws.sigs.k8s.io |
 | **Docker**\*    | CAPD                           | Infrastructure           | https://cluster-api.sigs.k8s.io |
 | **Addon Provider Fleet**    | CAAPF                           | Addon           | http://github.com/rancher-sandbox/cluster-api-addon-provider-fleet |
+| **vSphere**         | CAPV                           | Infrastructure           | https://github.com/kubernetes-sigs/cluster-api-provider-vsphere |
+| **Azure** (Only AKS clusters)         | CAPZ                           | Infrastructure           | https://capz.sigs.k8s.io/ |
```

Comment:
can we list providers in the same group in sequence (add these 2 under CAPD basically)?